ui/src/components/SampleView/SampleImage.jsx

```diff
@@ -209,7 +209,7 @@ export default class SampleImage extends React.Component {
   setVCellSpacing(e) {
     let value = Number.parseFloat(e.target.value);
     if (Number.isNaN(value)) {
-      value = '';
+      value = 0;
```

Comment:
Can the cell spacing really be set to `0`? Would it make more sense to have an early return and not call `setCellSpace` or `setCurrentCellSpace` at all maybe? Just wondering :shrug:

Reply:
I think cell spacing `0` is valid value. It means you don't have any extra space between your cells, when you do your mesh grid scans.

Reply:
Yes 0 is normally a valid value. Meaning no spacing.